jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/java/JavaRuntime.java

```diff
@@ -64,7 +64,21 @@ public default String getSystemProperty(String key) throws DataUnavailable, Corr
      * @throws CorruptDataException
      */
     public JavaVMInitArgs getJavaVMInitArgs() throws DataUnavailable, CorruptDataException;
-    
+
+/*[IF CRIU_SUPPORT]*/
```

Comment:
This is not appropriate. A build without CRIU support should be able to analyze a system dump from a VM with CRIU support.

Reply:
Removed the `[IF CRIU_SUPPORT]` blocks.